src/zimscraperlib/zim/metadata.py

```diff
@@ -1,22 +1,139 @@
 from __future__ import annotations
 
+import base64
 import datetime
 import re
 from collections.abc import Iterable
-from typing import Any
+from dataclasses import dataclass
+from typing import Any, NamedTuple
 
 import regex
 
-from zimscraperlib.constants import (
-    ILLUSTRATIONS_METADATA_RE,
-    MANDATORY_ZIM_METADATA_KEYS,
-    MAXIMUM_DESCRIPTION_METADATA_LENGTH,
-    MAXIMUM_LONG_DESCRIPTION_METADATA_LENGTH,
-    RECOMMENDED_MAX_TITLE_LENGTH,
-)
 from zimscraperlib.i18n import is_valid_iso_639_3
 from zimscraperlib.image.probing import is_valid_image
 
+RECOMMENDED_MAX_TITLE_LENGTH = 30
+MAXIMUM_DESCRIPTION_METADATA_LENGTH = 80
+MAXIMUM_LONG_DESCRIPTION_METADATA_LENGTH = 4000
+
+ILLUSTRATIONS_METADATA_RE = re.compile(
+    r"^Illustration_(?P<height>\d+)x(?P<width>\d+)@(?P<scale>\d+)$"
+)
+
+RawMetadataValue = (
```

Comment:
What would you recommend? The fact that it is a type and not a value itself should not be reflected in the name (we do not name all types `...Type`). Would it be better to use `ScraperMetadataValue` here instead of `RawMetadataValue`, and `LibzimMetadataValue` (or just `bytes`) instead of `CleanMetadataValue`? I think it is important to indicate this type is only for the metadata value, and not the whole metadata (name + value). We could as well use `Content` instead of `Value` if it helps.